civis/io/_files.py

```diff
@@ -235,6 +238,16 @@ def file_to_civis(buf, name, api_key=None, client=None, **kwargs):
     Small or non-seekable file-like objects will be uploaded with a
     single post.
     """
+    if isinstance(buf, str):
```

Comment:
You should check against `six.string_types` instead of `str`, for Python 2 compatibility.